.github/workflows/alpine.yml

```diff
@@ -0,0 +1,33 @@
+name: Build bindings for Alpine releases
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: node:${{ matrix.node }}-alpine
+      volumes:
+        - $GITHUB_WORKSPACE/vendor:/__w/node-sass/node-sass/vendor
+      env:
+        SKIP_SASS_BINARY_DOWNLOAD_FOR_CI: true
+    strategy:
+      matrix:
+        node:
+          - 6
+          - 8.16.2 # Something in 8.17.0 is breaking NPM cache install
+          - 10
```

Comment:
Give https://github.com/sass/node-sass/issues/2822 we'll want to use 10.16

Reply:
I didn't run into this one in the testing, can you give an example of what Alpine version you hit the issue with?

Reply:
In the end the gist of the problem appears to be that binaries built on newer alpine version didn't work on older alpine versions. I think most alpine docker images come 3.9, 3.10, 3.11, so we'll probably want to build binaries in whatever the lowest is for the specific node version.

Reply:
OK, there are tags that include the Alpine version, so I can take a look at aligning that details to the run

Reply:
I've pinned them to the oldest versions now